### PR TITLE
Add macosicons extension

### PR DESCRIPTION
**Pull request by shldk**
## Description

Extension for browsing and applying icons from macOSicons.com

#### `Icons Browser`

home

The process of updating an icon for the app is very easy:

1. When you find an icon you like, select it, press `return` _(or select the first item from "Actions" menu)_

2. Select the application you want to change the icon for

3. And _viola,_ your new icon is immediately applied system-wide!

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/macosicons/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "macosicons",
+  "title": "MacOSIcons.com",
```
Comment (pernielsentikaer) [inline extensions/macosicons/package.json:4]:
```suggestion
  "title": "MacOSIcons",
```
Could we use this name instead?

File: extensions/macosicons/src/components/actions/icon-actions.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      toast.style = Toast.Style.Success;
+      toast.title = `${app.name} updated, might've caused screen to flicker once`;
+    } catch (e) {
+      toast.style = Toast.Style.Failure;
+
+      toast.title = e?.toString() ?? "Something went wrong";
+    }
+  };
+  return (
+    <ActionPanel.Submenu title="Save As New Icon For" icon={Icon.Highlight}>
```
Comment (pernielsentikaer) [inline extensions/macosicons/src/components/actions/icon-actions.tsx:43]:
```suggestion
    <ActionPanel.Submenu title="Save as New Icon For" icon={Icon.Highlight}>
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

– Add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) for setting the icon and include information on how to revert it back, I couldn't find an easy way to do that.

- Remove /command-icon.png

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by Visual-Studio-Coder** [comment]
hmm. I really was looking forward to this extension. (did you possibly forget to click the request re-review button?)

**Comment by 3llomi** [comment]
Is there any chance for this to get reviewed?

**Comment by Visual-Studio-Coder** [comment]
Yeah, just request rereview by clicking that button 🙏🏼  🙏🏼  🙏🏼  🙏🏼  dude I seriously need this extension

**Comment by pernielsentikaer** [comment]
It's still awaiting @shldk to make the corrections, and we have to figure out some things. Nothing has changed since my last review, as far as I can see.

**Pull request closed without merging**